velox/exec/tests/SpillTest.cpp

```diff
@@ -42,12 +42,21 @@ class SpillTest : public testing::Test,
       int64_t targetFileSize,
       int numPartitions,
       int numBatches,
+      int numDuplicates,
       int64_t expectedNumSpilledFiles) {
     auto tempDirectory = exec::test::TempDirectoryPath::create();
     const std::string kSpillPath = tempDirectory->path + "/test";
     std::shared_ptr<FileSystem> fs;
     std::vector<std::string> spilledFiles;
     const int kNumRowsPerBatch = 100'000;
+    std::vector<std::vector<int64_t>> values(numBatches);
+    std::vector<int64_t> expectedValues(kNumRowsPerBatch * numBatches * 2);
```

Comment:
Are we creating numBatches batches of kNumRowsPerBatch * 2 rows each? kNumRowsPerBatch suggests that it is the number of rows in the batch, but it looks like the actual number of rows is double of that.

Reply:
Yeah, it is a little bit confusing.